code/modules/busy_space/systems.dm

```diff
@@ -0,0 +1,602 @@
+		new /datum/lore/location("the Prometheus in Vounna", list(ATC_TYPICAL, ATC_DIPLO)),
+		new /datum/lore/location("the remains of the SGWP Delilah in Vounna", list(ATC_SALVAGE)),
+		new /datum/lore/location("the Sisyphus in Vounna", list(ATC_TYPICAL)),
+		)
+
+/datum/lore/system/exalts_light
+	name = "Exalt's Light"
+	desc = "But everyone just calls it after its planet. A theocratic technocratic autocracy run by Angessa Martei, it lead the push for Almachi independence and has remained recalcitrant ever since."
+	planets = list("Angessa's Pearl")
+	locations = list(
+		new /datum/lore/location("Skylight on Angessa's Pearl", list(ATC_ALL_CIV, ATC_DIPLO)),
+		new /datum/lore/location("Melt on Angessa's Pearl", list(ATC_TRANS, ATC_FREIGHT)), // melt doesn't get emergency responders
+		new /datum/lore/location("Angessa's Grave on Angessa's Pearl", list(ATC_TRANS, ATC_LUX, ATC_MED)),
+		new /datum/lore/location("Tsunami on Angessa's Pearl", list(ATC_TYPICAL)),
+		new /datum/lore/location("a demonstration on Angessa's Pearl", list(ATC_DEF, ATC_MED)),
+		new /datum/lore/location("a Far Kingdoms wreck in Exalt's Light", list(ATC_SALVAGE, ATC_SCI)),
+		)
+
+//Five Arrows
+//might need lore ppl to weigh in here
+
+/datum/lore/system/new_seoul
+	name = "New Seoul"
+	desc = "The capital system of Five Arrows. Historically wealthy."
+	locations = list(
+		new /datum/lore/location("the Five Arrows capital in New Seoul", list(ATC_ALL_CIV, ATC_DIPLO)),
+		new /datum/lore/location("NSAIST main campus in New Seoul", list(ATC_SCI)),
+		)
+
+/datum/lore/system/sidhe
+	name = "Sidhe"
+	desc = "A university and mining town in the Heights, decimated by the Incursion and subsequent Ue'Katish invasions. An ongoing refugee crisis."
+	planets = list("Abhartach")
+	locations = list(
+		new /datum/lore/location("Daoine Institute in Sidhe", list(ATC_TYPICAL, ATC_SCI)),
+		new /datum/lore/location("Sidhe Alsace Fleet Base", list(ATC_TYPICAL)), //it's not really a permenant installation so def and med missions there make sense
+		new /datum/lore/location("Qux'Xinu on Abhartach, Sidhe", list(ATC_ALL_CIV, ATC_DIPLO)),
+		new /datum/lore/location("Colony Impossible in Sidhe", list(ATC_TYPICAL)),
+		new /datum/lore/location("the remains of the Colony Possible in Sidhe", list(ATC_SALVAGE)),
+		new /datum/lore/location("the School of Tachyon Physics in Sidhe", list(ATC_DEF, ATC_SCI)),
+		new /datum/lore/location("the XIS Hawthorn in Sidhe", list(ATC_FREIGHT, ATC_TRANS, ATC_DEF)),
+		new /datum/lore/location("Xe'xi in Sidhe", list(ATC_TYPICAL)),
+		)
+
+/datum/lore/system/kauqxum
+	name = "Kauq'xum"
+	desc = "The Skrellian monarchies closest to Sol. Joined the Five Arrows for mutual defense."
+
+/datum/lore/system/mahimahi
+	name = "Mahi-Mahi"
+	desc = "Primarily an organic agricultural world, the planet is divided into two distinct economic and cultural sectors."
+	planets = list("the planet") //its the same dumb thing as Love
```

Comment:
I believe Doctress was working on a thing for Mahi-Mahi and MIGHT have some location names. Though as of last draft a lot of them were placeholder "Northlandia" type things so idk! May be worth checking.